packages/frontend/pages/index.vue

```diff
@@ -18,57 +13,89 @@ GNU Affero General Public License for more details.
+                    </form>
+                    <div style="margin-bottom: 20px; width: 60%; width: 230px;" >
+                        <button-component class="button" buttonText="Create Record" backgroundColor="#CCECFD" onclick="window.location.href='/gdt'"
+                            borderColor="#CCECFD" color="#1E2019" padding="18px 22px" margin="0px 0px 0px 0px"></button-component>
+                    </div>
+                    <div id="homeTrackAssetDiv" style="visibility: hidden;">
+                        <TrackAsset inputWidth="75%"></TrackAsset>
+                    </div>
+                </div>
+                
+            </div>
+
+        </div>
+
+        <div class="row bg-frost" id="second-row" >
+            <div class="row rowtest">
+                <div class="col test" id="second-row-cols" v-for="item in second_row">
+                    <h3 class="text-iris" id="second-row-cols-h3">{{item.title}}</h3>
+                    <p class="text-eggplant" style="font-weight: 400;">{{ item.descr }}</p>         
+                </div>
+            </div>
+            <div class="col" style="text-align: center; margin-top: 50px;">
+                <button-component class="button" buttonText="About Us" color="#322253" onclick="window.location.href='about'"
+                    backgroundColor="#ffffff00"></button-component>
+            </div>
+
+        </div> 
+
+        <Learn_more id="learn-more"></Learn_more>
+
     </div>
- </template>
+
+</template>
+
+
 
 <script lang="ts">
+import Learn_more from '~/layouts/learn_more.vue';
+import ButtonComponent from '~/components/ButtonComponent.vue';
+let showTrack = false;
+
+const second_row = [
+    { title: "Simplicity & Accessibility", descr: "We believe that open-source projects should be simple to use and understand."},
+    { title: "Data Ownership", descr: "We do not have access to any user data, ensuring complete privacy and independent ownership."},
+    { title: "Open Source", descr:"Our projects are created for the public good and are available free of charge."}
+];
 
 export default {
-    
     methods: {
-        toggleView() {
-            const toggle = document.getElementById("toggle") as HTMLInputElement;
-            const createRecord = document.getElementById("create_record");
-            const createGroup = document.getElementById("create_group");
-            if (toggle.checked) {
-                createRecord.style.display = "none";
-                createGroup.style.display = "block";
-            } else {
-                createRecord.style.display = "block";
-                createGroup.style.display = "none";
+        // Function to have the 'Track an asset' input field appear
+        async trackingForm() {
+            let trackAssetDiv = <HTMLDivElement>document.getElementById("homeTrackAssetDiv");
+            let trackButton = <HTMLDivElement>document.getElementById("homeTrackButton");
```

Comment:
This works but we should refrain from accessing the DOM directly and try find a way to do this using Vue.

Reply:
I didn't work on this part originally, so I am struggling to find a way to do this with Vue. For the sake of getting higher priority issues completed I would maybe make a new issue for this.

Reply:
OK, can you please create the issue?

Reply:
Both requested issues have been created.